README.md

```diff
@@ -1,8 +1,15 @@
-# Fossify Gallery
+<div align="center">
 
 <img alt="Logo" src="graphics/icon.webp" width="120" />
 
-<a href="https://f-droid.org/en/packages/org.fossify.gallery/"><img src="https://fdroid.gitlab.io/artwork/badge/get-it-on-en.svg" alt="Get it on F-Droid" height=80/></a> <a href="https://apt.izzysoft.de/fdroid/index/apk/org.fossify.gallery"><img src="https://gitlab.com/IzzyOnDroid/repo/-/raw/master/assets/IzzyOnDroid.png" alt="Get it on IzzyOnDroid" height=80/></a>
+# Fossify Gallery
+
+### Browse your memories without any interruptions with this photo and video gallery
```

Comment:
Don't use headers for formatting, they have a semantic value. Also, you shouldn't jump in headers hierarchy like this. After the first level comes the second level etc.

If you want to read more: https://www.wearediagram.com/blog/how-not-to-use-html-headings (it's about HTML not Markdown, but it gets transformed to HTML anyway).